examples/audio/main.go

```diff
@@ -420,7 +420,7 @@ func (g *Game) Update() error {
 			panic("not reached")
 		}
 
-		g.musicPlayer.Close()
+		_ = g.musicPlayer.Close()
```

Comment:
I've fixed this, so please rebase this PR

Reply:
rebased :+1: